src/lib/select/select.spec.ts

```diff
@@ -173,6 +173,25 @@ describe('MdSelect', () => {
       expect(optionInstances[2].selected).toBe(false);
     });
 
+    it('should remove selection if option has been removed', () => {
+      let select = fixture.componentInstance.select;
+
+      trigger.click();
+      fixture.detectChanges();
+
+      let firstOption = overlayContainerElement.querySelectorAll('md-option')[0] as HTMLElement;
+
+      firstOption.click();
+      fixture.detectChanges();
+
+      expect(select.selected).toBe(select.options.first);
```

Comment:
Can you add fallback messages for both assertions if the test fails?  e.g. `Expected first option to be selected.`